mirrord/layer/src/common.rs

```diff
@@ -16,6 +20,12 @@ use crate::{
     HOOK_SENDER,
 };
 
+#[cfg(target_os = "macos")]
+lazy_static! {
+    /// Path of current executable, None if fetching failed.
+    pub static ref CURRENT_EXE: Option<String> = std::env::current_exe().ok().map(|path_buf| path_buf.to_string_lossy().to_string());
+}
```

Comment:
Can we not use https://docs.rs/once_cell/latest/once_cell/sync/struct.Lazy.html here